modules/javafx.graphics/src/main/java/javafx/scene/layout/Region.java

```diff
@@ -299,7 +301,7 @@ private static double scaledFloor(double value, double scale) {
     }
 
     private static double scaledCeil(double value, double scale) {
```

Comment:
Can you explicitly add here that this method guarantees 
`scaledCeil(scaledCeil(val, scale), scale) == scaledCeil (val)` 

It seems other code already assumed this, but it should not be taken for granted, so now that your fix assures this is the case, it can be taken for granted.

Reply:
Ok, make sense, I will add that.